fe/fe-common/src/test/java/org/apache/doris/common/property/PropertySchemaTest.java

```diff
@@ -244,8 +246,8 @@ public void testDatePropInvalidString() {
         exceptionRule.expect(IllegalArgumentException.class);
         exceptionRule.expectMessage(Matchers.containsString("Invalid time format"));
 
-        PropertySchema.DateProperty prop = new PropertySchema.DateProperty("key", new SimpleDateFormat("yyyy-MM-dd "
-                + "HH:mm:ss"));
+        PropertySchema.DateProperty prop = new PropertySchema.DateProperty("key", DateTimeFormatter.ofPattern("yyyy-MM-dd "
+            + "HH:mm:ss").withZone(ZoneId.systemDefault()));
```

Comment:
```
Error:  /home/runner/work/doris/doris/fe/fe-common/src/test/java/org/apache/doris/common/property/PropertySchemaTest.java:250:13: '+' has incorrect indentation level 12, expected level should be 16. [Indentation]
Error:  /home/runner/work/doris/doris/fe/fe-common/src/test/java/org/apache/doris/common/property/PropertySchemaTest.java:260:13: '+' has incorrect indentation level 12, expected level should be 16. [Indentation]
```

Reply:
code style

Reply:
@morningman ok, i will improve code style and solve these problems.

Reply:
@morningman thanks your time, i have solve these problems.